actors/evm/src/interpreter/precompiles.rs

```diff
@@ -42,17 +42,17 @@ pub struct Precompiles<BS, RT>(PhantomData<BS>, PhantomData<RT>);
 
 impl<BS: Blockstore, RT: Runtime<BS>> Precompiles<BS, RT> {
     const PRECOMPILES: [PrecompileFn<RT>; 9] = gen_precompiles();
-    const MAX_PRECOMPILE: H160 = {
-        let mut bytes = [0u8; 20];
-        bytes[0] = Self::PRECOMPILES.len() as u8;
-        H160(bytes)
+    const MAX_PRECOMPILE: U256 = {
+        let mut limbs = [0u64; 4];
+        limbs[0] = Self::PRECOMPILES.len() as u64;
+        U256(limbs)
```

Comment:
this is highly likely to be incorrect depending on if U256 is converted to LE encoding or not (H160 is BE encoded). Tests will smoke the answer out quickly though.

Reply:
ugh, indeed we will find out.

Reply:
its probably ok coz i am passing the bits directly though.